src/dmd/hdrgen.d

```diff
@@ -784,6 +784,12 @@ public:
         buf.writestring(t.dstring);
     }
 
+    override void visit(TypeTraits t)
```

Comment:
A similar visit method should be added in the `ParseTimeTransitiveVisitor` [1] otherwise the tree traversal will be truncated.

[1] https://github.com/dlang/dmd/blob/master/src/dmd/transitivevisitor.d